zerver/views/__init__.py

```diff
@@ -1127,10 +1127,6 @@ def update_active_status_backend(request, user_profile, status=REQ,
     return json_success(ret)
 
 @authenticated_json_post_view
-def json_update_active_status(request, user_profile):
-    return update_active_status_backend(request, user_profile)
-
-@authenticated_json_post_view
 def json_get_active_statuses(request, user_profile):
     return json_success(get_status_list(user_profile))
 
```

Comment:
`json_get_active_statuses` can probably also be migrated to a REST style API view (can be an additional commit); I think it should probably be get `GET /users/presence`